test/LEGO.AsyncAPI.Tests/Bindings/Kafka/KafkaBindings_Should.cs

```diff
@@ -11,7 +11,7 @@ namespace LEGO.AsyncAPI.Tests.Bindings.Kafka
     using NUnit.Framework;
     using System.Collections.Generic;
 
-    internal class KafkaBindings_Should
+    internal class KafkaBindings_Should : TestBase
```

Comment:
I could actually revert this `TestBase` change however I would like to leverage it in a future review.  My first attempt at this I added a `IAsserter` instance to the base class, but ended up just extending FluentAssertions instead.

Reply:
It doesnt look to be used. So perhaps we can remove it for now. 
Unless you need it for something specific.